mlflow/pyfunc/__init__.py

```diff
@@ -353,37 +353,47 @@ def spark_udf(spark, model_uri, result_type="double"):
 
     :param spark: A SparkSession object.
     :param model_uri: The location, in URI format, of the MLflow model with the
-                      :py:mod:`mlflow.pyfunc` flavor, for example:
+                      :py:mod:`mlflow.pyfunc` flavor. For example:
 
                       - ``/Users/me/path/to/local/model``
                       - ``relative/path/to/local/model``
                       - ``s3://my_bucket/path/to/model``
                       - ``runs:/<mlflow_run_id>/run-relative/path/to/model``
 
-                      For more information about supported URI schemes, see the
-                      `Artifacts Documentation <https://www.mlflow.org/docs/latest/tracking.html#
-                      supported-artifact-stores>`_.
+                      For more information about supported URI schemes, see
+                      `Artifact Stores <https://www.mlflow.org/docs/latest/tracking.html#
```

Comment:
Based on above suggestion about renaming the `Artifact Store Locations` to `Referencing Artifacts`, this link should be renamed.